src/components/order-service/order-service.controller.ts

```diff
@@ -176,7 +179,8 @@ export class OrderServiceController implements angular.IController {
        },
        spec: {
          serviceClassName: serviceClassName,
-         planName: this.ctrl.selectedPlan.name
+         planName: this.ctrl.selectedPlan.name,
+         parameters: this.ctrl.parameterData
```

Comment:
depending on when this gets in might need to update this to the new parameter model format before sending